### PR TITLE
typos - one set of minor changes

### PR DESCRIPTION
**Pull request by mdkessler**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [ ] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/02.intro.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ Although deep learning is a distinct subfield of machine learning, it is still a
 It is subject to the many limitations inherent to machine learning, and many best practices for machine learning also apply to deep learning.
 In addition, as with all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
 Ultimately, the tips we collate range from high-level guidance to best practices for implementation, and it is our hope that they will provide actionable, deep learning-specific instruction for both new and experienced deep learning practitioners.
-By making deep learning more accessible for use in biological research, we aim to improve the overall usage and reporting quality of deep learning in the literature, and to enable increasing numbers of researchers to utilize these state-of-the art techniques effectively and accurately.
\ No newline at end of file
+By making deep learning more accessible for use in biological research, we aim to improve the overall usage and reporting quality of deep learning in the literature, and to enable increasing numbers of researchers to effectively and accurately utilize these state-of-the art techniques.
```
Comment (agitter) [inline content/02.intro.md:27]: This creates a split infinitive

File: content/03.when-to-use-dl.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ## Tip 1: Decide whether deep learning is appropriate for your problem {#appropriate}
 
 In recent years, the number of publications implementing deep learning in biology have risen tremendously.
-Given deep learning's usefulness across a range of scientific questions and data modalities, it may appear that it is a panacea for modeling problems.
-Indeed, neural networks are universal function approximators, meaning that they are in principle capable of learning any function [@doi:10.1007/BF02551274; @tag:hornik-approximation].
-If deep learning is so powerful and popular, why would one ever not choose to use it?
+This is likely driven by deep learning's usefulness across a range of scientific questions and data modalities, and can contribute to the appearance of deep learning as a panacea for modeling problems.
```
Comment (agitter) [inline content/03.when-to-use-dl.md:4]:
```suggestion
This trend is likely driven by deep learning's usefulness across a range of scientific questions and data modalities and can contribute to the appearance of deep learning as a panacea for modeling problems.
```

**Review by agitter — CHANGES_REQUESTED** [review]
Thanks for these improvements.  I have two small grammatical suggestions.

Edit: I noticed there are merge conflicts that will have to be addressed.

File: content/02.intro.md (hunk 1 above)
Comment (mdkessler) [reply content/02.intro.md:27]: I tend to ascribe to the Star Trek approach ("to boldly go"), and like split infinitives sometimes :)

But happy to change it!

Thanks for the other fix...I tried to "commit it", but my apologies if I did that wrong and created a merge conflict.

**Comment by mdkessler** [comment]
Forget this pull request - I forgot to update fork and it created merge conflicts. I'm fixing it on my end and will resend with all changes.

**Pull request closed without merging**